mne/io/eyelink/eyelink.py

```diff
@@ -284,6 +287,45 @@ def _find_overlaps(df, max_time=0.05):
     return ovrlp.drop(columns=tmp_cols).reset_index(drop=True)
 
 
+@fill_doc
+def read_eyelink_calibration(
+    filename, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Return info on calibrations collected in an eyelink file.
+
+    Parameters
+    ----------
+    filename : str
+        Path to the eyelink file (.asc).
+    screen_size : tuple
+        The width and height (in meters) of the screen that the eyetracking
+        data was collected with. For example (.531, .298) for a monitor with
```

Comment:
```suggestion
        data was collected with. For example ``(.531, .298)`` for a monitor with
```

Double backticks around variables improve readability of the documentation.